src/main.py

```diff
@@ -281,14 +284,31 @@ def _run_testing(env: BaseEnv, approach: BaseApproach) -> Metrics:
         else:
             monitor = None
         try:
-            traj, execution_metrics = utils.run_policy(
-                policy,
-                env,
-                "test",
-                test_task_idx,
-                task.goal_holds,
-                max_num_steps=CFG.horizon,
-                monitor=monitor)
+            if CFG.env == "behavior" and \
```

Comment:
this is okay for now given the impending deadline, but the accumulating behavior special-casing in the code base is making me sad face. after the deadline, let's talk about making a persistent branch or fork of predicators and moving all of the behavior stuff into it

Reply:
"We will do" -Willie